seshdash/views.py

```diff
@@ -584,15 +584,16 @@ def get_notifications_alerts(request):
 
     arr = []
 
-
+    
     for site in sites:
-          arr.append({
-            "site":site.site_name,
-            "counter":Sesh_Alert.objects.filter(isSilence=False,site=site).count(),
-            "alerts_counter":Sesh_Alert.objects.filter(isSilence=False).count(),
-            "site_id":site.id,
-            })
-
+        if(Sesh_Alert.objects.filter(isSilence=False ,site=site).count() != 0):
+              arr.append({
+                "site":site.site_name,
+                "counter":Sesh_Alert.objects.filter(isSilence=False,site=site).count(),
+                "alerts_counter":Sesh_Alert.objects.filter(isSilence=False).count(),
```

Comment:
Why don't you calculate the total number of alerts in the JS?
Should be faster and DRY